src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -321,29 +323,165 @@ export default class TaskRoleContainerList extends React.Component {
     }
   }
 
-  getColumns() {
-    const columns = [
+  getColumns(showDebugInfo) {
+    const optionalColumns = [
       {
-        key: 'number',
-        name: 'No.',
+        key: 'accountableRetries',
+        name: 'Accountable Retries',
         headerClassName: FontClassNames.medium,
-        minWidth: 50,
-        maxWidth: 50,
+        minWidth: 150,
+        maxWidth: 200,
         isResizable: true,
         onRender: (item, idx) => {
           return (
-            !isNil(idx) && (
-              <div className={FontClassNames.mediumPlus}>{idx}</div>
-            )
+            <div className={FontClassNames.mediumPlus}>
+              {item.accountableRetries}
+            </div>
           );
         },
       },
       {
-        key: 'name',
-        name: 'Container ID',
+        key: 'startTime',
+        name: 'Start Time',
+        headerClassName: FontClassNames.medium,
+        minWidth: 150,
+        maxWidth: 200,
+        isResizable: true,
+        onRender: item => {
+          return (
+            <div className={c(FontClassNames.mediumPlus)}>
+              {printDateTime(DateTime.fromMillis(item.createdTime))}
+            </div>
+          );
+        },
+      },
+      {
+        key: 'completionTime',
+        name: 'Completion Time',
+        headerClassName: FontClassNames.medium,
+        minWidth: 150,
+        maxWidth: 200,
+        isResizable: true,
+        onRender: item => {
+          return (
+            <div className={c(FontClassNames.mediumPlus)}>
+              {printDateTime(DateTime.fromMillis(item.completedTime))}
+            </div>
+          );
+        },
+      },
+      {
+        key: 'currentAttemptLaunchedTime',
+        name: 'Current Attempt Launched Time',
```

Comment:
Move currentAttemptLaunchedTime/currentAttemptCompletedTime between createdTime and completionTime?

Reply:
![image](https://user-images.githubusercontent.com/8444268/86736798-63b19100-c066-11ea-8ec3-873ebd3d6b88.png)